src/Resources/app/storefront/src/js/filter-category-select.plugin.js

```diff
@@ -170,7 +170,7 @@ export default class FilterCategorySelectPlugin extends FilterBasePlugin {
         const subCategory = icon !== null ? icon.nextElementSibling : null;
 
         if (!checkbox.checked) {
-            const subCategories = checkbox.parentNode.getElementsByClassName('sub-item');
+            const subCategories = checkbox.parentNode.querySelectorAll('sub-item');
```

Comment:
`sub-item` is an element? Shouldn't this be `.sub-item` if you want to access element by class? And how is this working with a wrong selector? Please check this.

Reply:
@D3xter7 How was this passing before if it was using the wrong selector? That is what I had asked in the previous comment.